libs/entitlement/src/main/java/org/elasticsearch/entitlement/runtime/policy/PolicyParserUtils.java

```diff
@@ -54,9 +94,15 @@ public static Map<String, Policy> createPluginPolicies(Collection<PluginData> pl
             String pluginName = pluginRoot.getFileName().toString();
             final Set<String> moduleNames = getModuleNames(pluginRoot, entry.isModular());
 
-            var overriddenPolicy = parsePolicyOverrideIfExists(overrides, version, entry.isExternalPlugin(), pluginName, moduleNames);
-            if (overriddenPolicy.isPresent()) {
-                pluginPolicies.put(pluginName, overriddenPolicy.get());
+            var overriddenPolicy = parsePolicyOverrideIfExists(
+                overrides.get(pluginName),
```

Comment:
Is it ok to keep using the term "override" here instead of "patch"?

Reply:
IDK - I'd like to have all uniform, but which one do we like more?
(I think we should go with patch, if the intention is to use the merge strategy for plugin policies too)

Reply:
Agreed, "override" implies you're ignoring what's there.